docs/concepts.md

```diff
@@ -46,4 +46,30 @@ A Canary deployment exposes a subset of users to the new version of the applicat
 [![How Canary deployments work](concepts-assets/canary-deployments.png)](concepts-assets/canary-deployments.png)
 
 The picture above shows a canary with two stages (10% and 33% of traffic goes to new version) but this is just an example. With Argo Rollouts you can define the exact number of stages
-and percentages of traffic according to your use case.
\ No newline at end of file
+and percentages of traffic according to your use case.
+
+## Which strategy to choose
+
+In general Blue/Green is the easier strategy to start with, but also the more limited. We recommend you start with Blue/Green deployments first and as you gain confidence for your metrics and applications switch to Canaries.
+
+You also need to examine if your application can handle canaries or not.
+
+* Blue/Green always works because only one application is active at a time. Not all applications can have different versions running in parallel at the same time (which is what canaries are doing)
+* Blue/Green is simpler because it works WITHOUT a traffic manager. Not all people like service meshes, and not all people have an ingress that is supported by Argo Rollouts
+* Blue/Green also works with services that use queues and databases (workers that fetch tasks)
+
+Here is a summary table for the two approaches.
+
+|                           |         Blue/Green         | Canary         |
```

Comment:
```suggestion
|                           |         Blue/Green         | Canary  w/Service Mesh       |
```

Reply:
True. But the big blocker is that Canary needs your application to have 2 instances that are both receiving live traffic at the same time. And not all  apps can do this. The traffic manager is important but completely unrelated to that requirement.

Reply:
Fair. I guess we rank that a bit different (in terms of ease). but i guess there's a big bag of gotchas/tradeoffs, and I get this is meant as an intro guide, so I'll try to avoid getting bogged in details.